profiles/dheerajjha451.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/106474979/244020746-bbd43201-43f3-466c-a85d-2378156a6217.png",
+    "issueId": 1839,
+    "name": "Dheeraj Jha",
+    "username": "Dheerajjha451"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

@Panquesito7  I had made the changes; you can check them now.